src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Components/SortableHeaderCell/Default.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@using Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Models
+@model Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Components.SortableHeaderCell.SortableHeaderCellViewModel
+
+<a href="@Model.GetSortUrl()"
+   style="white-space: nowrap">
+    @Model.DisplayName
+
+    @if (Model.IsActive && Model.SortDirection != null)
```

Comment:
Model.IsActive could be removed.

Reply:
If removed the glyphs will be shown for all columns.